workers/ohsome_quality_analyst/utils/helper.py

```diff
@@ -181,3 +182,9 @@ def load_sklearn_model(path: str):
     model = joblib.load(path)
     warnings.resetwarnings()
     return model
+
+
+async def sem_task(task: Coroutine, semaphore=asyncio.Semaphore(4)) -> Coroutine:
```

Comment:
Should this function be part of the `helper.py` or the `oqt.py` module?

Reply:
I think it's fine to define such a independent helper function here. It is not specific to the functionality in `oqt.py`.